manim/config/logger.py

```diff
@@ -114,7 +114,7 @@ def set_file_logger(log_file_path):
 logger = logging.getLogger("manim")
 # The console is set to None as it will be changed by set_rich_logger.
 console = None
-
+install()
```

Comment:
```suggestion
# Install the rich traceback subpackage. This is necessary because **reasons here**.
install()
```

Reply:
Weird that it requires an "install" but whatever I guess. Does this "install" take long?

Reply:
> Does this "install" take long?

I don't think so.

But what it does is add *beautiful* tracebacks. See screenshots.